wajibisha/templates/homepage_country.html

```diff
@@ -0,0 +1,104 @@
+        <img src="{% static 'img/presentation.svg' %}" class="presentation-img">
+      </div>
+    </div>
+  </div>
+{% endblock header_content %}`
+
+{% block content_container %}
+  <div class="showcase how-it-works text-center">
+    {% block how_it_works %}
+    <div class="wrapper clearfix">
+      <h4>SOME OF THE PROMISES WE'RE TRACKING</h4>
+      <br/>
+      <!--Start of cards-->
+      <div class="card-group">
+        <div class="card p-0 m-3 body-text">
+          <div class="pt-4">
+            <img class="card-img-top rounded mx-auto d-block" src="{% static 'img/tracker/category/housing.svg' %}" alt="Card image cap"></div>
+            <div class="card-block">
+              <h5 class="card-title">HOUSING</h5>
+            </div>
+            <div class="details card-block text-left pt-3">
+              <p class="pl-4">Country: <b>Nairobi</b></p>
+              <p class="pl-4">Total no. of promises: <b>{{ housing_total }}</b></p>
+              <p class="pl-4" >Promises fulfilled: <b>{{ housing_fulfilled }}</b></p>
+            </div>
+          </div>
+          <div class="card p-0 m-3 body-text">
+            <div class="pt-4">
+              <img class="card-img-top rounded mx-auto d-block" src="{% static 'img/tracker/category//health.svg' %}" alt="Card image cap"></div>
+              <div class="card-block">
+                <h5 class="card-title">HEALTH</h5>
+              </div>
+              <div class="details card-block text-left pt-3">
+                <p class="pl-4">Country: <b>Nairobi </b></p>
+                <p class="pl-4">Total no. of promises: <b>{{ health_total }}</b></p>
+                <p class="pl-4">Promises fulfilled:<b>{{ health_fulfilled }}</b></p>
+              </div>
+            </div>
+            <div class="card p-0 m-3 body-text">
+              <div class="pt-4">
+                <img class="card-img-top rounded mx-auto d-block" src="{% static 'img/tracker/category/jobs.svg' %}" alt="Card image cap"></div>
+                <div class="card-block">
+                  <h5 class="card-title">JOBS</h5>
+                </div>
+                <div class="details card-block text-left pt-3">
+                  <p class="pl-4">Country: <b>Nairobi</b></p>
+                  <p class="pl-4">Total no. of promises: <b>{{ jobs_total }}</b></p>
+                  <p class="pl-4">Promises fulfilled: <b>{{ jobs_fulfilled }}</b></p>
+                </div>
+              </div>
+            </div>
+          </div>
+          {% endblock %}
+  </div>
+
+  <script>
+      function selectCounty() {
+          var selectedCounty = document.getElementById("county_select").value;
+          if (selectedCounty === 'nairobi') {
+              window.location.href = '/profiles/county-47-nairobi/';
+          }
+
+      }
+  </script>
+
+  {% include 'blocks/mission.html' %}
+  {% include 'blocks/tweets.html' %}
+  {% include 'blocks/newsletter.html' %}
+{% endblock %}
+
+
+
```

Comment:
Why not have this directly has the `homepage.html` template instead of extending it from there?